src/cpp/shared/game/game.cpp

```diff
@@ -194,26 +194,8 @@ void game::SetRandomHeroArmies(int heroIdx, int isAI) {
   int tierChance;
 
   hasTier[TIER_ONE] = 1;
-  if (isAI) {
-    tierChance = 80;
-  } else {
-    tierChance = 50;
-  }
-  if (Random(0, 99) < tierChance) {
-    hasTier[TIER_TWO] = 1;
-  } else {
-    if (isAI) {
-      tierChance = 35;
-    } else {
-      tierChance = 75;
-    }
-    hasTier[TIER_TWO] = Random(0, 99) < tierChance;
-  }
-  
-  /*hasTier[TIER_TWO] = Random(0, 99) < (isAI < 1 ? 50 : 80); // If isAI, there is a ~80% chance that this hero will get Tier 2 creatures; if not, then a ~50% chance
-  if ((isAI < 1 ? 25 : 65) <= Random(0, 99)) { // This line is only useful if the previous line doesn't assign "true" to hasTier[TIER_TWO].
-    hasTier[TIER_TWO] = 1;                     // If isAI, there is a ~35% chance that this hero will get Tier 2 creatures; if not, then a ~75% chance
-  }*/
+  hasTier[TIER_TWO] = Random(0, 99) < 87;
```

Comment:
Gorgeous.

Reply:
While I agree, I find that it doesn't take much to look gorgeous compared to what used to be there xD